### PR TITLE
#160522376  ch-mocha-test -view-all-orders 

### PR DESCRIPTION
**Pull request by Johnpaul-Borree**
#### What does this PR do?
Test api endpoint  GET /api/v1/orders
#### Description of Task to be completed?
- write tests for endpoint
- Ensure all tests pass
- look up continuous integration changes with test
#### How should this be manually tested?
- run command  npm run test
#### What are the relevant pivotal tracker stories?
#160522376 
#### Screenshots (if appropriate)
screenshot 16

### CODE REVIEW

File: src/controller/ordersControllers.js [hunk 1]
```diff
@@ -1,37 +1,40 @@
 import Joi from 'joi';
 import orders from '../models/orders';
 
+export function getAllOrders (req, res) {
+	res.status(200).json({status: 'Success', Orders: orders});
+}
 
 export function postOrder (req, res) {
```
Comment (KaiserPhemi) [inline src/controller/ordersControllers.js]: #### Critical
Same feedback on #11

File: test/ordersTests.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+				.end((err, res) => {
+					res.should.have.status(200);
+					res.body.should.be.a('object');
+					res.body.should.have.property('Orders').be.a('array');
+					res.body.should.have.property('status').eql('Success');
+					done();
+				});
+		});
+	});
+});
```
Comment (KaiserPhemi) [inline test/ordersTests.js]: #### Nitpick
Add a new line

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
Work on feedback

File: test/ordersTests.js (hunk 2 above)
Comment (Johnpaul-Borree) [reply test/ordersTests.js]: Finished implementing feedback

**Comment by Johnpaul-Borree** [comment]
merging view all orders mocha test to develop

**Pull request merged**